README.md

```diff
@@ -373,15 +373,22 @@ will give a more in-depth description of the command and available options.
 #### Proxy Submissions
 
 The autograder allows course staff to submit on behalf of students using "proxy" submissions.
-The proxy submit command adds two parameters to the normal submission command:
+The proxy submit commands add two parameters to the normal submission command:
 
  - `proxy-email` -- The student that the course staff is submitting on behalf of.
  - `proxy-time` -- The "fudged" submission time that is automatically set to the earlier time between now and one minute before the due date.
    - If you want to set the time manually, time is the number of milliseconds from the [Unix Epoch](https://en.wikipedia.org/wiki/Unix_time).
 
+In future examples with `proxy-time`, we can use a [Unix Epoch time converter](https://www.epochconverter.com/) to change the target time
+of March, 4th 2025 at 12:00 PM Pacific Time to milliseconds from the Unix Epoch, `1741118400000`.
+
 As the commands come from course staff, the submissions are not subject to submission restrictions.
```

Comment:
s/come from/are issued by/
s/the submissions/the proxy submissions/

I just want to make this a little more specific.